rs/sns/swap/src/swap.rs

```diff
@@ -894,9 +894,24 @@ impl Swap {
         // for all NF investors.
         let mut global_cf_memo: u64 = NEURON_BASKET_MEMO_RANGE_START;
         for cf_participant in self.cf_participants.iter_mut() {
+            let controller = cf_participant.try_get_controller();
```

Comment:
Note to reviewer: This must be done outside of the loop or we get complains from borrowck